packages/graphql-validate-fixtures/test/__snapshots__/validate.test.ts.snap

```diff
@@ -1179,7 +1179,7 @@ Object {
   "validationErrors": Array [
     Object {
       "keyPath": "generation",
-      "message": "value does not match enum Generation (available values: FIRST, SECOND, THIRD)",
+      "message": "Value \\"SEVENTH\\" does not exist in \\"Generation\\" enum.",
```

Comment:
these snapshots changed more because they come from the try / catch we added (they used to be a custom error)